Splunk.OTel.DotNet.psm1

```diff
@@ -0,0 +1,375 @@
+﻿#Requires -RunAsAdministrator
+
+function Get-Current-InstallDir() {
+    return [System.Environment]::GetEnvironmentVariable("OTEL_DOTNET_AUTO_INSTALL_DIR", [System.EnvironmentVariableTarget]::Machine)
```

Comment:
Minor: maybe `SPLUNK_OTEL_DOTNET_AUTO_INSTALL_DIR` in order to offer side-by-side installation? 🤔